deepmd/pt_expt/infer/deep_eval.py

```
@@ -0,0 +1,460 @@
+    neighbor_list : ase.neighborlist.NewPrimitiveNeighborList, optional
+        The ASE neighbor list class to produce the neighbor list. If None, the
+        neighbor list will be built natively in the model.
+    **kwargs : dict
+        Keyword arguments.
+    """
+
+    def __init__(
+        self,
+        model_file: str,
+        output_def: ModelOutputDef,
+        *args: Any,
```

Ruff (0.15.2)

[warning] Unused method argument: `args`

(ARG002)